README.md

```diff
@@ -7,17 +7,19 @@
 
 **Made with 💝 for <img src=".github/tux.png" align="top" width="24" alt="Tux (Linux)"/>**
 </div>
+
 <p align="center">
   &nbsp;<a href="https://wimpysworld.io/discord" target="_blank"><img alt="Discord" src="https://img.shields.io/discord/712850672223125565?style=for-the-badge&logo=discord&logoColor=%23ffffff&label=Discord&labelColor=%234253e8&color=%23e4e2e2"></a>&nbsp;
   &nbsp;<a href="https://fosstodon.org/@wimpy" target="_blank"><img alt="Mastodon" src="https://img.shields.io/badge/Mastodon-6468fa?style=for-the-badge&logo=mastodon&logoColor=%23ffffff"></a>&nbsp;
   &nbsp;<a href="https://twitter.com/m_wimpress" target="_blank"><img alt="Twitter" src="https://img.shields.io/badge/Twitter-303030?style=for-the-badge&logo=x&logoColor=%23ffffff"></a>&nbsp;
   &nbsp;<a href="https://linkedin.com/in/martinwimpress" target="_blank"><img alt="LinkedIn" src="https://img.shields.io/badge/LinkedIn-1667be?style=for-the-badge&logo=linkedin&logoColor=%23ffffff"></a>&nbsp;
 </p>
 
+
 # Introduction
 
 **Quickemu** is a wrapper for the excellent [QEMU](https://www.qemu.org/) that
-automatically *"does the right thing"* when creating virtual machines. No
+automatically *“does the right thing”* when creating virtual machines. No
```

Comment:
These funky quote are back :slightly_smiling_face:

Reply:
These come from using `gfm` as the output format. I think I turn them off if they offend you. The secret is to turn of the smart extension on the markdown input if not using markdown output ( so `... -f markdown-smart -t gfm ...`

``` text

# from the pandoc manual
Extension: smart
       Interpret straight quotes as curly quotes, --- as em-dashes, -- as en-dashes, and ... as ellipses.  Nonbreaking spaces are inserted after certain abbreviations, such as “Mr.”

       This extension can be enabled/disabled for the following formats:

       input formats
              markdown, commonmark, latex, mediawiki, org, rst, twiki, html

       output formats
              markdown, latex, context, rst

       enabled by default in
              markdown, latex, context (both input and output)

       Note: If you are writing Markdown, then the smart extension has the reverse effect: what would have been curly quotes comes out straight.
```

I rather approve of the "smart" behaviour, but if you prefer not I'll spin up a regen without.  I'm looking at what this `markdown-smart` option does for the man page generation. Also I'll need to tweak the Makefiles once we're settled on the required incantations.